pyproject.toml

```diff
@@ -12,19 +12,44 @@ packages = [
 python = ">=3.8,<4"
 django = ">=3.2"
 ruff = "^0.0.292"
+typing-extensions = {version = ">=4.0.1", python = "3.11"}
```

Comment:
we also need to add deps to setup.cfg

this file is used mainly in development (if my memory serves me well)